templates/snippetEditor.jst

```diff
@@ -3,36 +3,43 @@
 		<h3 class="snippet-editor__heading snippet-editor__heading-icon snippet-editor__heading-icon-eye"><%- i18n.snippetPreview %></h3>
     <p class="screen-reader-text"><%- i18n.snippetPreviewDescription %></p>
 
-        <div class="snippet_container snippet-editor__container" id="title_container">
-            <span class="screen-reader-text"><%- i18n.titleLabel %></span>
-            <span class="title" id="render_title_container">
-                <span id="snippet_title">
-                    <%- rendered.title %>
-                </span>
-            </span>
-            <span class="title" id="snippet_sitename"></span>
-        </div>
-        <div class="snippet_container snippet-editor__container" id="url_container">
-            <span class="screen-reader-text"><%- i18n.slugLabel %></span>
-            <cite class="url urlBase" id="snippet_citeBase">
-                <%- rendered.baseUrl %>
-            </cite>
-            <cite class="url" id="snippet_cite">
-                <%- rendered.snippetCite %>
-            </cite>
-            <span class="down_arrow"></span>
-        </div>
-        <div class="snippet_container snippet-editor__container" id="meta_container">
-            <span class="screen-reader-text"><%- i18n.metaDescriptionLabel %></span>
-            <% if ( "" !== metaDescriptionDate ) { %>
-                <span class="snippet-editor__date">
-                    <%- metaDescriptionDate %> -
-                </span>
-            <% } %>
-            <span class="desc" id="snippet_meta">
-                <%- rendered.meta %>
-            </span>
-        </div>
+		<div class="snippet-editor__view-toggle">
+			<button class="snippet-editor__view-icon snippet-editor__view-icon-desktop" data-type="desktop" />
+			<button class="snippet-editor__view-icon snippet-editor__view-icon-mobile" data-type="mobile" />
```

Comment:
The buttons are empty, there's nothing screen readers can announce.

![screen shot 2016-11-24 at 15 28 49](https://cloud.githubusercontent.com/assets/1682452/20623429/cea4e55e-b307-11e6-9a66-bbabd508f9cb.png)

I'd suggest to add some screen-reader-text or `aria-label` ("Desktop preview" - "Mobile preview" or something like that).

Also, there's no visual or audible feedback about which is the currently selected, active, view. For the visual feedback I guess we have to wait for a design proposal as mentioned in https://github.com/Yoast/yoast-components/issues/110
For the audible feedback, I'd propose to use the `aria-pressed` attribute, as done for example in the WordPress Customizer for the device preview buttons. This attribute uses false/true values and conveys the information the buttons are "toggle buttons", mutually exclusive. Meaning only one of them can have a "pressed" state. Screen readers will announce the buttons as "toggle button" and will announce also their state (some screen readers use the word "selected", others the word "pressed"):

![screen shot 2016-11-24 at 15 29 42](https://cloud.githubusercontent.com/assets/1682452/20623475/07df97e2-b308-11e6-882b-479e12eefec6.png)

![screen shot 2016-11-24 at 15 29 54](https://cloud.githubusercontent.com/assets/1682452/20623476/07e107b2-b308-11e6-8fea-9b5a2743ac77.png)